dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -1800,8 +1800,6 @@ paths:
               schema:
                 type: string
           description: An elastic search result.
-      security:
```

Comment:
A lot of endpoints are losing their security and bearer token information. Seems odd

Reply:
It's likely old version of yq installed

Reply:
I have 3.1.1, the code uses 3.1.2. It seems weird that that would break it.

Reply:
I've mentioned that at least 3 times during stand up and pretty much yelled it out that if you don't have 3.1.2 or higher, it will break